tests/Benchmark/RoutingBench.php

```diff
@@ -0,0 +1,30 @@
+<?php
+declare(strict_types=1);
+
+namespace Cake\Tests\Benchmark;
+
+use Cake\Routing\Route\Route;
+
+class RoutingBench
+{
+    protected Route $route;
+
+    public function __construct()
+    {
+    }
+
+    public function setUpCompile()
+    {
+        $this->route = new Route('/{controller}/{action}/*');
+    }
+
+    /**
+     * @BeforeMethods("setUpCompile")
```

Comment:
We should perhaps use PHP attributes instead of annotations and run the benchmark on PHP 8.